packages/api-sdk/src/medical/models/demographics.ts

```diff
@@ -46,19 +47,20 @@ export type PersonalIdentifier = z.infer<typeof personalIdentifierSchema>;
 
 export const genderAtBirthSchema = z.enum(["F", "M"]);
 
-const phoneLength = 10;
 export const contactSchema = z
   .object({
     phone: z.coerce
       .string()
-      .transform(phone => stripNonNumericChars(phone))
-      .refine(phone => phone.length === phoneLength, {
-        message: `Phone must be a string consisting of ${phoneLength} numbers. For example: 4153245540`,
+      .transform(phone => normalizePhoneNumber(phone))
```

Comment:
Generally I don't agree with the normalize then validate approach. We should validate explicitly based on what we think a valid phone number *is* -- they should have to enter this. i.e. we should not let them enter a phone number that "normalizes" to a valid phone number...

Reply:
Yeah, thinking about this again I think it makes sense so not modify the input at all. This is really changing the API contract, and can lead to other Apps behaving in a way that's more difficult to change later.

I initially thought it made sense b/c we were stripping non numeric chars and that's part of the normalization, but I agree we should not do anything more than that, I'll revert it here.

And, with this, then it makes sense to have normalization to always "autofix", then can remove that parameter and simplify the function.

Reply:
done